examples/dark-mode/app/utils/theme.server.ts

```diff
@@ -0,0 +1,34 @@
+import { createCookieSessionStorage } from "remix";
+
+import { Theme, isTheme } from "./theme-provider";
+
+// Make use to set the environment variable SESSION_SECRET before running the code
+const sessionSecret = process.env.SESSION_SECRET;
```

Comment:
You could set a fallback value for the example to work

Reply:
I've added a default value, as you suggested